entries/107_show_notes.bacf

```diff
@@ -0,0 +1,79 @@
+    <li><a href="https://github.com/apple/swift-evolution/blob/master/proposals/0235-add-result.md">Result in Swift 5</a></li>
+    <li><a href="https://github.com/apple/swift-evolution#primary-focus-abi-stability">Application Binary Interface (ABI) stability</a></li>
+    <li><a href="https://help.apple.com/xcode/mac/current/#/devbbdc5ce4f">App Thinning</a></li>
+    <li><a href="https://twitter.com/twostraws/status/1088507816841789440">Playgrounds "might not execute"</a></li>
+    <li><a href="https://twitter.com/nerdonica/status/1088459155264995328">Veronica Ray's Xcode tips and tricks Twitter thread</a></li>
+    <li><a href="https://www.youtube.com/playlist?list=PLuoeXyslFTuYQ9Hoh42Bw8sPYMlTOV0V7">Paul Hudson's "Xcode in 20 seconds" video series</a></li>
+    <li><a href="https://developer.apple.com/library/archive/documentation/IDEs/Conceptual/xcode_help-command_shortcuts/MenuCommands/MenuCommands014.html">Xcode Keyboard Shortcuts (⌃+⌘+R and - ⌘+⇧+J)</a></li>
+    <li><a href="https://developer.apple.com/documentation/xcode_release_notes/xcode_10_2_beta_release_notes/swift_5_release_notes_for_xcode_10_2_beta">Exclusive Memory Access at Runtime</a></li>
+    <li><a href="https://twitter.com/steipete/status/1088821790024105984">LLDB debugger's "v" (frame variable)</a></li>
+    <li><a href="https://developer.apple.com/documentation/xcode_release_notes/xcode_10_2_beta_release_notes/swift_5_release_notes_for_xcode_10_2_beta">Swift 3 support dropped</a></li>
+    <li><a href="https://www.youtube.com/watch?v=ePuOrCbIW-o">"Taken for Granted", Bas' talk at try! Swift NYC 2018</a></li>
+    <li><a href="https://twitter.com/bzamayo/status/1088766602814869504">Xcode Dark Mode</a></li>
+    <li><a href="https://github.com/apple/swift-evolution#primary-focus-abi-stability">Memory Ownership Model</a></li>
+    <li><a href="https://github.com/apple/swift-evolution#other-improvements">Async/Await</a></li>
+    <li><a href="https://gist.github.com/lattner/31ed37682ef1576b16bca1432ea9f782">Swift Concurrency Manifesto</a></li>
+  </ul>
+</p>
+<p>
+  <h3>The Swift Community Podcast</h3>
+
+  <ul>
+    <li><a href="https://www.swiftcommunitypodcast.org">The Swift Community Podcast website</a></li>
+    <li><a href="https://github.com/SwiftCommunityPodcast/podcast">The Swift Community Podcast on GitHub</a></li>
+    <li><a href="https://twitter.com/theswiftpodcast">The Swift Community Podcast on Twitter</a></li>
+    <li><a href="https://twitter.com/garricn">Garric Nahapetian</a></li>
+    <li><a href="https://twitter.com/johnsundell">John Sundell</a></li>
+    <li><a href="https://twitter.com/clattner_llvm">Chris Lattner</a></li>
+    <li><a href="https://swiftweekly.github.io">Swift Weekly Brief</a></li>
+    <li><a href="TBA">The Swift Community Podcast: Scaling A Codeless Open Source Swift Community with Bas Broek</a></li>
```

Comment:
All good now, except this one is missing. Should also regenerate the site afterwards.